sdk/trustedsigning/arm-trustedsigning/review/arm-trustedsigning-api-certificateProfiles-node.api.md

```diff
@@ -0,0 +1,51 @@
+## API Report File for "@azure/arm-trustedsigning"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type { Client } from '@azure-rest/core-client';
+import type { OperationOptions } from '@azure-rest/core-client';
+import type { OperationState } from '@azure/core-lro';
+import type { PollerLike } from '@azure/core-lro';
+
+// @public
+export function $delete(context: CodeSigningContext, resourceGroupName: string, accountName: string, profileName: string, options?: CertificateProfilesDeleteOptionalParams): PollerLike<OperationState<void>, void>;
```

Comment:
was this method autogenerated? Seems like a bug but maybe I just dont know - cc @joheredi